onnxruntime/test/providers/qnn/qnn_basic_test.cc

```diff
@@ -1113,6 +1113,12 @@ TEST_F(QnnHTPBackendTests, UseHtpSharedMemoryAllocatorForInputs) {
     qnn_ep = QnnExecutionProviderWithOptions(provider_options);
   } catch (const OnnxRuntimeException& e) {
     // handle particular exception that indicates that the libcdsprpc.so / dll can't be loaded
+    // NOTE: To run this on a local Windows ARM64 device, you need to copy libcdsprpc.dll to the build directory:
+    //  - Open File Explorer
+    //  - Go to C:\Windows\System32\DriverStore\FileRepository\
```

Comment:
Looks like linux compilers do not like the `\` in this comment. I'll fix it.
>/onnxruntime_src/onnxruntime/test/shared_lib/test_inference.cc:1965:5: error: multi-line comment [-Werror=comment]
 1965 |     //  - Go to C:\Windows\System32\DriverStore\FileRepository\